modules/client/src/internal/client/encoding.ts

```diff
@@ -199,7 +199,7 @@ export class ClientEncoding extends ClientCore implements IClientEncoding {
       throw new Error("Could not pin the metadata on IPFS");
     }
     const daoInterface = DAO__factory.createInterface();
-    const args = new TextEncoder().encode(cid);
```

Comment:
Should we add/modify the existing tests for this? We are going to split the encoding and the pinning, but at least the returned URI format should be checked for regressions

Reply:
the tests succeed. what would you want to modify?